p8e-api/src/main/kotlin/io/provenance/engine/service/AuthenticationService.kt

```diff
@@ -50,8 +51,9 @@ class AuthenticationService(
             return null
         }
 
+        // Verify signature against the auth public key we have on record.
         val verified = Signature.getInstance(Constant.JWT_ALGORITHM).apply {
-            initVerify(publicKey)
+            initVerify(affiliate.authPublicKey.toJavaPublicKey())
```

Comment:
Just confirming that we are just going to copy existing affiliate's signing public key over to the auth_public_key column, right?

Reply:
:yes